config/games/werewolfnight.ts

```diff
@@ -0,0 +1,43 @@
+import { ServerGame } from "../../types/types";
+import { postJson } from "../../utils/utils";
+
+const game: ServerGame = {
+    id: "werewolfnight",
+    name: "werewolf-night.com",
+    author: "Duc Ngo Viet",
```

Comment:
I suggest including the following “based on” information:

```suggestion
    author: "Duc Ngo Viet",
    basedOn: {
        game: "Mafia",
        author: "Dimitry Davidoff",
        link: "https://en.wikipedia.org/wiki/Mafia_(party_game)",
        bggId: 925,
    },
```

Reply:
thanks!